rj_gameplay/reg_pass_costing/cost_heuristic.py

```diff
@@ -0,0 +1,157 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+import numpy as np
+
+MAX_PASS_DIST = 5
+MAX_GOAL_DIST = 7.5
+MAX_AVG_OPP_DIST = 6
+MAX_NEAREST_OPP_DIST = 2.5
+MAX_OPP_BETWEEN_ROBOT_GOAL = 5
+MAX_OPP_BETWEEN_ROBOT_BALL = 1
+MIN_GOAL_DIST_BONUS = 1
+MAX_GOAL_DIST_BONUS = 1.75
```

Comment:
What are the points of these maximums? The logic behind them might be sound, but I'm just wondering.

Reply:
The point of the maximums was to set the range of values that maps to the cost. For example, robots that are more than 7.5 units away from the goal should all receive the maximum cost, and robots that have the nearest opponent more than 2.5 units away should all receive the minimum cost.